### DISCUSSION TITLE
Fab.com Library Assets

### DISCUSSION THREAD

**Discussion by vladimirgiraldo** [AchetaGames/Epic-Asset-Manager#290] [Q&A] [answered]
Hi. With the replacement of UE Marketplace with FAB, is there a roadmap or plan to update the Epics Asset Manager to include the content from fab.com library?

**Comment by aknarts** [comment, marked as answer]
That  depends on the exposure of the API https://github.com/AchetaGames/Epic-Asset-Manager/issues/289 and the usability of the official UE Fab Plugin, if it works on Linux and does everything that it should than the work of this project is pretty much done and it will be archived. 

If it does not, or it is a flaky experience, and the API is somewhat usable - i.e. I can reverse engineer it sure the support will be added.

As for roadmaps I have not had time to dedicate to the project lately so committing to something like that in terms of timeline:). I would sure like to get it done though.

**Reply by vladimirgiraldo** [reply]
Right... totally understandable... thks for answering.
